src/openvic-simulation/military/Unit.cpp

```diff
@@ -263,90 +263,41 @@ bool UnitManager::load_unit_file(GoodManager const& good_manager, ast::NodeCPtr
 
 		switch (type) {
-		case LAND:
-			{
+			case LAND: {
 				bool primary_culture = false;
 				std::string_view sprite_override, sprite_mount, sprite_mount_attach_node;
 				fixed_point_t reconnaissance = 0, attack = 0, defence = 0, discipline = 0, support = 0, maneuver = 0, siege = 0;
 
-				ret &= add_key_map_entries(key_map,
-					"primary_culture", ZERO_OR_ONE, expect_bool(assign_variable_callback(primary_culture)),
-					"sprite_override", ZERO_OR_ONE, expect_identifier(assign_variable_callback(sprite_override)),
-					"sprite_mount", ZERO_OR_ONE, expect_identifier(assign_variable_callback(sprite_mount)),
-					"sprite_mount_attach_node", ZERO_OR_ONE, expect_identifier(assign_variable_callback(sprite_mount_attach_node)),
-					"reconnaissance", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(reconnaissance)),
-					"attack", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(attack)),
-					"defence", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(defence)),
-					"discipline", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(discipline)),
-					"support", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(support)),
-					"maneuver", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(maneuver)),
-					"siege", ZERO_OR_ONE, expect_fixed_point(assign_variable_callback(siege))
-				);
+				ret &= add_key_map_entries(key_map, "primary_culture", ZERO_OR_ONE, expect_bool(assign_variable_callback(primary_culture)), "sprite_override", ZERO_OR_ONE, expect_identifier(assign_variable_callback(sprite_override)), "sprite_mount", ZERO_OR_ONE, expect_identifier(assign_variable_callback(sprite_mount)), "sprite_mount_attach_node", ZERO_OR_ONE, expect_identifier(assign_variable_callback(sprite_mount_attach_node)), "reconnaissance", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(reconnaissance)), "attack", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(attack)), "defence", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(defence)), "discipline", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(discipline)), "support", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(support)), "maneuver", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(maneuver)), "siege", ZERO_OR_ONE, expect_fixed_point(assign_variable_callback(siege)));
```

Comment:
Could this be reverted please, this being one line is absolutely a nightmare to read.